README.md

```diff
@@ -1,17 +1,242 @@
 # Review Service
 
-This repository contains a simple contract to prompt user to review the current application on Android and iOS.
+This repository contains simple contracts to prompt the user to review the current application if conditions are met based on provided data for Android and iOS.
```

Comment:
The goal of the service is to add business logic (conditions and data tracking) around a basic review prompter.